certgen/certProcessor/src/main/java/org/incredible/certProcessor/qrcode/utils/QRCodeImageGenerator.java

```diff
@@ -258,4 +256,18 @@ private static int getImageType(String colorModel) {
             return BufferedImage.TYPE_BYTE_GRAY;
         }
     }
+
+    private  InputStream getFontFile(List<File> files) throws FileNotFoundException {
+
+        Iterator<File> iterator = files.iterator();
+        InputStream fontStream=null;
+        while (iterator.hasNext()) {
+            File file = iterator.next();
+            if (file.getName().equals("Verdana.ttf")) {
+                fontStream = new FileInputStream(file);
+                logger.info("File Name++++++++++++++++++++=" + file.getName());
```

Comment:
Remove +++ put some message and no need to have two logger , use one logger and combine both message